src/modules/CaaSMapper.ts

```diff
@@ -180,8 +181,9 @@ export class CaaSMapper {
         const simpleValue: string | number = entry.value
         return simpleValue
       case 'CMS_INPUT_RADIOBUTTON':
-        // TODO: This should be mapped to interface Option
-        const radiobuttonOption = entry.value
+        const radiobuttonOption: Option | null = entry.value
```

Comment:
Customers might expect `entry.value` as a result. Your solution is good, but it's not downwards compatible.